plugins/modules/idrac_secure_boot.py

```diff
@@ -0,0 +1,387 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.6.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: idrac_secure_boot
+short_description: Configures the iDRAC secure boot
+version_added: "9.6.0"
+description:
+  - This module allows you to configure the iDRAC secure boot.
+extends_documentation_fragment:
+  - dellemc.openmanage.idrac_x_auth_options
+options:
+  import_certificates:
+    type: bool
+    description:
+        - Import all the specified key certificates.
+        - I(import_certificates) is C(true) either of I(platform_key) or I(KEK) or I(database) or I(disallow_database) is required.
```

Comment:
If I(import_certificates) is C(true), then the value of one or more of the following parameters is required:  I(platform_key), or I(KEK), or I(database), or I(disallow_database).

Reply:
we can not keep : in middle of docs, sanity fails.

Any other suggestion